test/helpers/testdefs/StructureDefiniton-geolocation.json

```diff
@@ -0,0 +1,763 @@
+{
```

Comment:
I think this makes literally zero difference, but now that I've seen it, I can't un-see it. I think there's a typo in the file name, it should be `StructureDefinition-geolocation.json`.

Reply:
Oh man.

Reply:
It's like typo city today.  I'm embarrassed.  Fixed in ae365f3.